app/poros/core_stats.rb

```diff
@@ -0,0 +1,38 @@
+class CoreStats
+  attr_reader :stats
+
+  def initialize(ancestry_buffs)
+    @stats = {}
+    distribute_scores
+    distribute_buffs(ancestry_buffs)
+  end
+
+  def distribute_scores
+    scores = [15, 14, 13, 12, 10, 8].shuffle
```

Comment:
In the future I want to see if we can 'roll' to get _really_ random numbers in here